packages/common/src/downloads/index.ts

```diff
@@ -1 +1,4 @@
-export * from "./build-existing-exports-portal-query";
```

Comment:
Looks like you took the old module and split it up into new modules. That's fine as long as everything that was previously exported from the old module is still exported from the new ones (so that there are no breaking changes). I assume you were conscious of that, but I just want to confirm.

Reply:
Absolutely, you can see the updated exports associated with this on line 1 for the function, and line 2 for the `PORTAL_EXPORT_TYPES` const